.gitignore

```diff
@@ -18,6 +18,7 @@
 .env.test.local
 .env.production.local
 .env
+.eslintcache
```

Comment:
Co to za plik?

Reply:
czasem vscode czasem sobie cos cachuje lokalnie na komputerze i dodaje sie do projektu, dodałem żeby tego nie pushowac na serwer niepotrzebnie za kazdym razem